src/core/util.js

```diff
@@ -283,4 +285,221 @@ var parseTimeSeriesForC3 = function(graph_data) {
   };
 }
 
-module.exports = { parseDataForC3, parseTimeSeriesForC3, dataApiToC3 }
+/*
+    Takes a multistats object of the following form and 
+    1) flattens it, and
+    2) rounds numeric values 
+    for passing to the DataTable component for rendering:
+
+    {
+      "tasmin_Amon_CanESM2_historical_r1i1p1_19610101-19901231": 
+      {
+        "median": 278.34326171875,
+        "min": 225.05545043945312,
+        "units": "K",
+        "mean": 273.56732177734375,
+        "max": 303.601318359375,
+        "ncells": 8192,
+        "stdev": 22.509726901403784,
+        "run": "r1i1p1"
+      },
+      "tasmin_Amon_CanESM2_historical_r1i1p1_19710101-20001231": 
+      {
+        ...
+      }
+    };
+*/
+var parseBootstrapTableData = function(data) {
+    var flatData = [];
+    var model_count = 0;
+    for (let model in data) {
+        var year_range_re = new RegExp("[0-9]{8}","g");
+        var year_range = [];
+        var lastIndex = 0;
+        while (year_range_re.test(model)){
+            year_range_re.lastIndex = lastIndex;
+            year_range.push(year_range_re.exec(model)[0].slice(0,4));
+            lastIndex = year_range_re.lastIndex;
+        }
+        var period = year_range[0] + " - " + year_range[1];
+        var modelInfo = {
+            "model_period": period,
+            "run": data[model]['run'],
+            "min": +data[model]['min'].toFixed(PRECISION),
+            "max": +data[model]['max'].toFixed(PRECISION),
+            "w_mean": +data[model]['mean'].toFixed(PRECISION),
+            "median": +data[model]['median'].toFixed(PRECISION),
+            "w_stdev": +data[model]['stdev'].toFixed(PRECISION),
+            "units": data[model]['units']
+        };
+        flatData.push(modelInfo); 
+    }
+    return flatData;
+}
+
+/*
+    Helper function for exportTableDataToWorksheet, creates summary rows that appear at the top of the exported worksheet
+*/
+var createWorksheetSummaryCells = function(summary_data, time_of_year) {
+    // store worksheet cell contents to be later encoded as per output format
+    var cells = [];
+    var cell;
+    // write summary rows at top of worksheet
+    var summary_header = ["Model", "Emissions Scenario", "Time of Year", "Variable ID", "Variable Name"];
+    var summary_keys = ["model_id", "experiment", "time_of_year", "variable_id", "variable_name"];
+
+    var num_summary_rows = 3;
+    var num_summary_cols = summary_keys.length
+    for(var R = 0; R < num_summary_rows; ++R) {
+        for(var C = 0; C < num_summary_cols; ++C) {
+            if(R == 0) cell = {v: summary_header[C]};
+            else if(R == 1) {
+                if(summary_keys[C] == 'time_of_year') {
+                  cell = {v: time_of_year};      
+                }
+                else cell = {v: summary_data[summary_keys[C]]};
+            }
+            else cell = {v: ""};
```

Comment:
Always use enclosing `{}` even for one line blocks. Omitting them risks bugs later on.